Library/Homebrew/resource.rb

```diff
@@ -286,6 +286,27 @@ def verify_download_integrity(_filename)
     end
 
     def tab
+      tab = manifest_annotations["sh.brew.tab"]
+      raise Error, "Couldn't find tab from manifest." if tab.blank?
+
+      begin
+        JSON.parse(tab)
+      rescue JSON::ParserError
+        raise Error, "Couldn't parse tab JSON."
+      end
+    end
+
+    def bottle_size
+      manifest_annotations["sh.brew.bottle.size"].to_i
```

Comment:
```suggestion
      manifest_annotations.fetch("sh.brew.bottle.size").to_i
```

or

```suggestion
      manifest_annotations["sh.brew.bottle.size"]&.to_i
```

depending on whether `manifest_annotations` is supposed to always have that key.